vagrant/post-deploy.d/00-run-docker.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -eu
+
+PAAS_DIR=/tmp/paas-bootstrap
+
+sudo apt update && sudo apt install docker-compose -y
```

Comment:
We thought about CoreOS, but then decided to use most common and most default image so that we can avoid any possible (obscure) compatibility issues into the future and also to ensure image will be available for long time. We chose ubuntu 16.04 as its support is until 2020...